hathor/cli/run_node.py

```diff
@@ -232,10 +233,41 @@ def create_wallet():
         for description in args.listen:
             self.manager.add_listen_address(description)
 
-        self.start_manager()
+        self.start_manager(args)
         self.register_resources(args)
 
-    def start_manager(self) -> None:
+    def start_sentry_if_possible(self, args: Namespace) -> None:
+        """Start Sentry integration if possible."""
+        if not args.sentry:
+            return
+        self.log.info('Starting Sentry', dsn=args.sentry)
+        try:
+            import sentry_sdk
+        except ModuleNotFoundError:
+            self.log.error('ERROR: You must install the package \'sentry-sdk\' to enable sentry.')
+            sys.exit(-3)
+
+        import hathor
+        from hathor.conf import HathorSettings
+        settings = HathorSettings()
+        sentry_sdk.init(
+            dsn=args.sentry,
+            release=hathor.__version__,
+            environment=settings.NETWORK_NAME,
+        )
+
+        import logging
+
+        import structlog
+        from structlog_sentry import SentryProcessor
+        processors = structlog.get_config()['processors']
+        processors.append(SentryProcessor(level=logging.ERROR))
+        structlog.configure(
+            processors=processors
```

Comment:
Are we able to send the exception StackTrace to Sentry with this setup?

By reading this I was left with the impression that maybe we won't be sending important information to Sentry, like the StackTrace, since we are putting SentryProcessor after the `format_exc_info` processor.
> When you want to use structlog's built-in format_exc_info processor, make that the SentryProcessor comes before format_exc_info! Otherwise, the SentryProcessor won't have an exc_info to work with, because it's removed from the event by format_exc_info.

https://github.com/kiwicom/structlog-sentry#usage

Reply:
Nice catch. It's fixed but it can't capture the stack trace in some cases (e.g. exception inside a deferred).